src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
you can use destructuring here to make your code more clear
import { string } from 'prop-types';